app/src/main/java/com/android/unio/model/event/EventViewModel.kt

```diff
@@ -51,6 +51,7 @@ constructor(private val repository: EventRepository, private val imageRepository
   fun loadEvents() {
     repository.getEvents(
         onSuccess = { eventList ->
+          eventList.forEach { event -> event.organisers.requestAll() }
           _events.value = eventList // Update the state flow with the loaded events
```

Comment:
Don't forget to remove these ChatGPT comments

Reply:
Not my doing haha, but yes I'm deleting it